guidelines/content-guidelines/content-strategy.md

```diff
@@ -62,7 +63,6 @@ Each technical topic must have the following document types:
 ### Optional
 
 You can add the following document type to the Kyma documentation:
```

Comment:
And  maybe we should add info here that these should go after the obligatory types?